src/xrSound/SoundRender_Emitter.h

```diff
@@ -85,7 +85,7 @@ class CSoundRender_Emitter final : public CSound_emitter
     std::atomic<Task*> prefill_task{};
 
     size_t current_block{};
-    int filled_blocks{};
+    u32 filled_blocks{};
```

Comment:
This was actually made as `int` on purpose in case of underflows.
It's better to leave it as is for now and examine the code that operates on this variable to make sure there are no underflows.
```suggestion
    int filled_blocks{};
```

Reply:
Alright, comment added because I have the memory of a goldfish

Reply:
No comment here, please.
Other person is working on xrSound currently and it might cause conflicts.

Reply:
Ok